core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/jobs/RemoteJob.java

```diff
@@ -119,6 +123,7 @@ protected final IStatus run(IProgressMonitor monitor) {
 					.build();
 			}
 			
+			LOG.error("Error encountered while executing remote job", e);
```

Comment:
Some identifying information about the job should be added to the message. `id`, `description`, class name of `request` are all good candidates.

Reply:
+1 for that